docs/contribution_guide/index.rst

```diff
@@ -0,0 +1,18 @@
+.. _contribution_guide:
+
+Contribution Guide
+==================
+
+This project welcomes contributions and suggestions. For information on our
+community, such as our :ref:`code_of_conduct`, :ref:`communication`, and
+regular :ref:`developer_calls`, refer to the :ref:`community` pages.
+
```

Comment:
How about something like:
```
This project welcomes contributions and suggestions.
We follow `GitHub community guidelines <https://...>`.
To find out how to connect to our community of users and developers,
read our :ref:`community pages <community>`. Here, we describe how
to structure your contributions.
```
Also, I noticed that when I click on "Community", it loads the Community page (as it should) and scrolls it up a bit (as it probably shouldn't). I'm not quite sure why this happens. [Not a major issue though.]

Reply:
I'd like to avoid putting the same code of conduct reference in multiple pages because it could get out of sync. I understand why you're rewriting it, and I've mostly adopted it. LMK if you want another change.

Reply:
I'm okay to leave as is. We can revisit in future. The rationale for my proposal was that our section on "Code of conduct" is just the link out, so it seems silly to send the user to our "code of conduct", just so they can click another link.